src/containers/BellScheduleContainer.js

```diff
@@ -146,7 +146,6 @@ class BellScheduleContainer extends React.PureComponent<Props, State> {
       );
 
       const now = this.state.refreshed;
-      now.subtract(7, 'hours');
```

Comment:
This is useful for debugging when school isn't in session. It still lets time progress so you can still see the blue bar slowly tick, compared to setting the time like I did in previous commits.